BUILD.bazel

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DISCLAIMER: Bazel support is community-based. The maintainers do not
+# use Bazel internally. The Bazel build can have security risks or
+# optimization gaps.
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "tbb12",
+    srcs = glob([
+        "include/oneapi/tbb/*.h",
+        "include/oneapi/tbb/detail/*.h",
+        "include/tbb/*.h",
+        "src/tbb/*.cpp",
+        "src/tbb/*.h",
+    ]) + [
+        "include/oneapi/tbb.h",
+    ] + select({
+        "@platforms//cpu:x86_64": glob(["src/tbb/tools_api/**/*.h"]),
+        "//conditions:default": [],
+    }),
+    hdrs = glob([
+        "include/tbb/*.h",
+    ]) + select({
+        "@platforms//cpu:x86_64": [
+            "src/tbb/tools_api/ittnotify_static.c",
+        ],
+        "//conditions:default": [],
+    }),
+    copts = [
+        "-w",
+    ],
+    defines =
+        select({
+            "@platforms//cpu:x86_64": [],
+            "//conditions:default": [
+                "USE_PTHREAD",
+            ],
+        }),
+    includes = [
+        "include",
+    ],
+    linkopts =
+        select({
+            "@bazel_tools//platforms:windows": [],
+            "//conditions:default": [
+                "-ldl",
+                "-pthread",
+                "-lrt",
+            ],
+        }),
+    local_defines = select({
+        "@platforms//cpu:x86_64": [
+            "__TBB_USE_ITT_NOTIFY",
+        ],
+        "//conditions:default": [],
+    }) + [
+        "__TBB_BUILD",
+    ],
+)
+
+'''
+cc_library(
+    name = "tbbmalloc",
+    srcs = [
```

Comment:
What about `glob` here?

Reply:
CMakeLists.txt does also not use globs here

Reply:
Why CMakeLists has hardcoded list: https://cmake.org/cmake/help/latest/command/file.html#glob
> **Note** We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.